### PR TITLE
Healthcare messaging - Do Not Merge Yet

### PR DESCRIPTION
**Pull request by mphprogrammer**
- added provisions for Healthcare api
- added va_healthcare_messaging gem
- added heathcare messaging controller
- added triage team controller
- added triage team serializers
- added specs for triage team serializers

I need a sanity check @saneshark @ayaleloehr - aka am I on the right path?

### CODE REVIEW

File: app/controllers/healthcare_messaging/v1/triage_teams_controller.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module HealthcareMessaging
```
Comment (saneshark) [inline app/controllers/healthcare_messaging/v1/triage_teams_controller.rb]: i think we're going to just dump these in v0 for now -- see my open PR.

**Comment by saneshark** [comment]
@mphprogrammer I did a quick first pass and other the one comment, I don't see any problems. We will likely improve and refactor on this... not seeing a route or request specs yet but imagine those are on the way.

**Comment by mphprogrammer** [comment]
Thanks - the specs will come next - I needed to break off for a bit. I will swing back to the specs in a bit. Thanks for the quick turn-around!

**Comment by mphprogrammer** [comment]
Deleting pr putting up new one.

**Pull request closed without merging**